pkgs/development/python-modules/opnieuw/default.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "opnieuw";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-BxhSz+AsL7lU3zw9tIL2R+Pxktb8NG2/UKPtEDJT+Qo=";
+  };
+
+  propagatedBuildInputs = [ typing-extensions ];
```

```suggestion
  build-system = [ setuptools ];

  dependencies = [ typing-extensions ];
```
required for pyproject